pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -290,6 +291,8 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo) (*apps.Dep
 				k8sutil.AppAttr:     appName,
 				k8sutil.ClusterAttr: c.Namespace,
 				osdLabelKey:         fmt.Sprintf("%d", osd.ID),
+				failureDomainKey:    failureDomainValue,
```

Comment:
How about factoring out a `getLabels()` method so we can apply these to both the deployment and pod template spec (line 312) consistently?

Reply:
done